autoload/ale/lsp/tsserver_message.vim

```diff
@@ -77,3 +80,23 @@ function! ale#lsp#tsserver_message#Quickinfo(buffer, line, column) abort
     \   'file': expand('#' . a:buffer . ':p'),
     \}]
 endfunction
+
+function! ale#lsp#tsserver_message#Rename(buffer, line, column) abort
+    return [0, 'ts@rename', {
+    \   'line': a:line,
+    \   'offset': a:column,
+    \   'file': expand('#' . a:buffer . ':p'),
+    \   'arguments': {'findInComments': v:true, 'findInStrings': v:true}
```

Comment:
Make finding names in comments and strings optional, and turn both off by default. They can be fuzzy and hit some false positives.